x-pack/plugins/cases/public/components/system_actions/cases/cases_params.tsx

```diff
@@ -193,7 +193,7 @@ export const CasesParamsFieldsComponent: React.FunctionComponent<
       <EuiFormRow
         fullWidth
         id="timeWindow"
-        error={errors.timeWindow}
+        error={errors.timeWindow as string[]}
```

Comment:
A bunch of dummy casting is caused by this alerting error object that is typed as 

```
export interface RuleFormParamsErrors {
  [key: string]: string | string[] | RuleFormParamsErrors;
}
```


not sure if there is a better short term fix :(

Reply:
This should be fine, in practice this is only ever `string[]` 👍